nodes/solid/polygon_face.py

```diff
@@ -43,9 +43,9 @@ def make_faces(self, verts, face_idxs):
         for face_i in face_idxs:
             face_i = list(face_i)
             face_i.append(face_i[0])
-            verts = [verts[idx] for idx in face_i]
-            verts = [Base.Vector(*vert) for vert in verts]
-            wire = Part.makePolygon(verts)
+            fc_verts = [verts[idx] for idx in face_i]
+            fc_verts = [Base.Vector(*vert) for vert in fc_verts]
```

Comment:
it may have a little performance boost if you define  `base_vector = Base.Vector` outside of both loops or with `from Base import Vector`